docs/docs/integrations/vectorstores/milvus.ipynb

```diff
@@ -100,10 +108,13 @@
    },
    "outputs": [],
    "source": [
+    "URI = \"http://127.0.0.1:19530\"  # For Milvus service started from docker, replace the IP address with your own.\n",
```

Comment:
URI = "./milvus_demo.db"  # The easiest way is to use Milvus Lite where everything is stored in a local file. If you have a Milvus server you can use the server URI such as "http://localhost:19530".